src/markdown-pages/why-galasa.md

```diff
@@ -0,0 +1,124 @@
+---
+path: "/about/different"
+title: "Why Galasa is different"
+---
+
+What makes Galasa different to other testing tools?
+
+If you've ever struggled to implement automated testing across a complex technology stack, you might recognize some of the issues that we identified during our design process. Expand the following headings to see how Galasa solves these challenges.
+
+## Running consistent, repeatable, reliable tests across the application stack: 
+
+
+<details>
+<summary><b>Ensuring consistent testing across technologies</b></summary>
+
+Automating tests effectively and consistently across multiple environments and multiple tools requires a framework that can support and integrate across the whole application stack. The same test must also be capable of running against multiple z/OS environments without code changes.
+
+Galasa is designed to work in hybrid cloud applications, operating with the entire application stack, so you can  provision in a cloud environment, but bind to a z/OS test environment.
+The Galasa framework works across multiple technologies in a consistent way, providing a platform to test different aspects of the application stack in the same language and in the same fashion. One test case can interact with 3270 emulators, Selenium, JMeter, batch jobs and other applications. 
```

Comment:
The initial statement is a little non-descript. Rather than "works across multiple techonolgies", perhaps re-phrase to "....integrates and wrappers multiple techonolgies and test tooling...."